src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -305,4 +305,16 @@
       it { expect(flash[:error]).to eq('Incident projects shall only create below maintenance projects.') }
     end
   end
+
+  describe 'GET #linking_projects' do
+    before do
+      login user_moi
+      apache2_project
+      another_project.projects_linked << apache_project
+      xhr :get, :linking_projects, project: apache_project
+    end
+
```

Comment:
Mmmm in what way it improves the test? I mean,  we don't usually test status code, it make sense for instance at show action that the status returned depends on a param... but generally I don't know if this worth. If it does then we should add it in almost every spec we did, right?